docs/_news/news08.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: news
+title: New for Codewind 0.8.0
+description: New for Codewind 0.8.0
+keywords: news, new, updates, update, version, hybrid
+duration: 1 minute
+permalink: news07
+---
+
+## Codewind 0.8.0
+Thursday 23 January 2020
+
+#### ✨ New Features and Highlights for 0.8.0 ✨
+
+**Appsody**
+- The `codewind-appsody-extension` supports passing environment options to the Appsody CLI using [`--docker-options`](https://github.com/eclipse/codewind/issues/1401).
+
+**Che**
+- The [`codewind-che-plugin` supports Helm 3](https://github.com/eclipse/codewind/issues/1320), and the insecure non-default tiller was removed.
+
+**IDEs (Both Eclipse and VS Code)**
+- Detailed project application status and errors are displayed in the UI in [Eclipse](https://github.com/eclipse/codewind-eclipse/issues/236) and [VS Code](https://github.com/eclipse/codewind/issues/1391).
+
+**VS Code**
+- Push registry is no longer required unless a [Codewind-style template source is enabled](https://github.com/eclipse/codewind/issues/1469).
+- If you [switch push registries, a warning appears](https://github.com/eclipse/codewind/issues/1421) that your namespace will be lost.
+- You can [cancel image registry creation during the `namespace` step, and you can use the `Esc` key to cancel the whole process](https://github.com/eclipse/codewind/issues/1621).
+- You can navigate the [remote connection settings page with keyboard shortcuts](https://github.com/eclipse/codewind/issues/1562).
+
+
+#### List of Fixes
+- Updated node.js express server generator to 4.2.2 in [Eclipse](https://github.com/eclipse/codewind-openapi-eclipse/pull/83) and [VS Code](https://github.com/eclipse/codewind-openapi-vscode/pull/77)
```

Comment:
Oops...might have overlooked this earlier. From what I can tell, "Node.js Express" should be capitalized like so because it's a name.